add docs for listing tokens (#226)
gobeyondidentity/developer-docs: docs/workflows/api-token.md

4. Check timestamps in JWT claims where `nbf` <= current time as unix timestamp in seconds <= `exp`
5. Check that JWT claims target tenant `bi_t` and target realm `bi_r` match the tenant and realm for the given application.
6. Check that JWT claims has the expected scopes.

## Listing Access Tokens

Our API also provides the functionality to list active tokens that are held by a given identity or an application.
In order to list tokens, you need to know the ID of the application that was used to issue the token.

You need to use a Management API Bearer token with the `tokens:read` scope in order to call this API.

### Request: Listing access tokens for an identity

In this case, the application is the issuer of the token and the identity is the subject of the token.

> <MultiLanguageCodeBlock
> curl='
> curl "https://api-${REGION}.beyondidentity.run/v1/tenants/${TENANT_ID}/realms/${REALM_ID}/tokens?application=${APPLICATION_ID}&principal_type=identity&principal_id=${IDENTITY_ID}"
> -H "Authorization Bearer ${MANAGEMENT_API_TOKEN}"
> ' title="/tokens"
> />

Reviewer: ```suggestion
<MultiLanguageCodeBlock
curl='curl "https://api-$(REGION).beyondidentity.run/v1/tenants/$(TENANT_ID)/realms/$(REALM_ID)/tokens?application=$(APPLICATION_ID)&principal_type=identity&principal_id=$(IDENTITY_ID)" \
-H "Authorization Bearer $(MANAGEMENT_API_TOKEN)"' 
title="/tokens"
/>
```

### Example Response

```json
{
  "tokens": [
    {
      "token_id": "uZcs8hF4_vR69eonor3U_lottYxtSfrX",
      "application": "43e65451-1a38-4e17-8fd3-ae5b582fd9c7",
      "scope": "",
      "exp": 1675177245,
      "iat": 1675090845,
      "token_type": "access",
      "principal_path": "tenants/0001fb357ac698bb/realms/d7e6e18f570421ce/identities/ce9d40cf7285a260",
      "token_format": "self_contained"
    }
  ]
}

> 

Reviewer: Removing extra space
```suggestion
```

```

### Request: Listing access tokens for an application

In this case, the application is both the issuer and the subject of the token,
which is why it needs to be in the request twice.

> <MultiLanguageCodeBlock
> curl='
> curl "https://api-${REGION}.beyondidentity.run/v1/tenants/${TENANT_ID}/realms/${REALM_ID}/tokens?application=${APPLICATION_ID}&principal_type=application&principal_id=${APPLICATION_ID}"
> -H "Authorization Bearer ${MANAGEMENT_API_TOKEN}"
> ' title="/tokens"
> />

Reviewer: And here too
```suggestion
<MultiLanguageCodeBlock
curl='curl "https://api-$(REGION).beyondidentity.run/v1/tenants/$(TENANT_ID)/realms/$(REALM_ID)/tokens?application=$(APPLICATION_ID)&principal_type=application&principal_id=$(APPLICATION_ID)" \
-H "Authorization Bearer $(MANAGEMENT_API_TOKEN)"' 
title="/tokens"
/>
```

### Example Response

```json
{
  "tokens": [
    {
      "token_id": "uZcs8hF4_vR69eonor3U_lottYxtSfrX",
      "application": "43e65451-1a38-4e17-8fd3-ae5b582fd9c7",
      "scope": "",
      "exp": 1675177245,
      "iat": 1675090845,
      "token_type": "access",
      "principal_path": "tenants/0001fb357ac698bb/realms/d7e6e18f570421ce/applications/43e65451-1a38-4e17-8fd3-ae5b582fd9c7",
      "token_format": "self_contained"
    }
  ]
}

> 

Reviewer: Removing extra space
```suggestion
```

4. Check timestamps in JWT claims where `nbf` <= current time as unix timestamp in seconds <= `exp`
5. Check that JWT claims target tenant `bi_t` and target realm `bi_r` match the tenant and realm for the given application.
6. Check that JWT claims has the expected scopes.

> ## Listing access tokens

Reviewer: For consistency with the other titles 
```suggestion
## Listing Access Tokens
```

Our API also provides the functionality to list active tokens that are held by a given identity or an application.
In order to list tokens, you need to know the ID of the application that was used to issue the token.

You need to use a Management API Bearer token with the `tokens:read` scope in order to call this API.

### Request: Listing access tokens for an identity

> ```bash

Reviewer: I was using https://curlconverter.com to create the other language code snippets from curl and then wrapping them in the <Tabs>. I can help with this part if you want or need.

Reviewer: I just updated this file with a `MultiLanguageCodeBlock` to handles this. If you merge main you should see how to add the curl to this block.

Author: Thank you so much, this is excellent!

curl 'https://api-${REGION}.beyondidentity.run/v1/tenants/${TENANT_ID}/realms/${REALM_ID}/tokens?application=${APPLICATION_ID}&principal_type=identity&principal_id=${IDENTITY_ID}'
-H "Authorization Bearer ${MANAGEMENT_API_TOKEN}"
```

> 

Reviewer 2: Can we add an example response here? I know they may look the same, but the doc is a bit confusing if we dont explicitly show them the response

Author: I added an example response to both cases. Would it make sense like this?

### Request: Listing access tokens for an application

```bash
curl 'https://api-${REGION}.beyondidentity.run/v1/tenants/${TENANT_ID}/realms/${REALM_ID}/tokens?application=${APPLICATION_ID}&principal_type=application&principal_id=${APPLICATION_ID}'
-H "Authorization Bearer ${MANAGEMENT_API_TOKEN}"
```

> In this case, the application is both the issuer and the subject of the token,
> which is why it needs to be in the request twice.

Reviewer: I would move this up to under the header & above the bash example 
I would also add a similar comment for the "Listing access tokens for an identity" as the `principal_id`

```suggestion
When listing access tokens for an application, the application need to be in the request twice as it is both the issuer `application` and the subject `principal_id` of the token. 
```

> ### Example response

Reviewer: If this is the same example response for both, you could add a comment here that the response for listing access tokens for either application or identity will return the same response.

Author: I added the response to both sections. I think that's least confusing as the responses are also a little bit different.

> ### Example response

Reviewer: ```suggestion
### Example Response
```
